.env

```diff
@@ -1 +1,2 @@
 SECRET_KEY_BASE=123
+BB_SECRET='this is a test secret'
```

Comment:
Agreed, but it was already in there when I started this work, so I had left it. I removed it now though.